packages/frontend/src/state/controller/hooks.ts

```diff
@@ -210,6 +210,17 @@ export const useGetTwapEthPrice = () => {
   return getTwapEthPrice
 }
 
+export const useGetTwapSqueethPrice = () => {
```

Comment:
we probably want a way to get a slot0 or twap with period of 1 price, as that is more useful for things like LP compositions.

The twap should be used for things like vault CR.

Reply:
This is already available as an atom `squeethPriceeAtom`